### DISCUSSION TITLE
Equirectangular images?

### DISCUSSION THREAD

**Discussion by rowild** [colmap/colmap#2294] [Q&A]
I wonder if Colmap is able to handle equirectangular images? My results so far are rather poor... (I did have awesome experiences with Metashape, though, and came to the conculsion that 360 images indeed are the best way to go to create point clouds.)

And if so, what do I have to look for? Could not find any tutorials.
And if not so - any plans to include this feature into Colmap?

Thank you!

**Comment by Jaz-yvr** [comment]
Having similar issue, selected the open cv fisheye camera type and selected data options to video frames, still got the issue of nan in the database, any different settings i should be trying for equilateral images?
